plugins/rtmp-services/data/services.json

```diff
@@ -1104,6 +1104,21 @@
                 "max video bitrate": 20000,
                 "max audio bitrate": 192
             }
+        },
+        {
+            "name": "Eventials",
```

Comment:
It is missing the following line:  "common": false,

Reply:
It's false by default, you don't have to add the value unless it's actually set to true.